source/mobile/build-custom-mobile-apps.rst

```diff
@@ -0,0 +1,84 @@
+
+Custom Mobile App Branding
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ensure you select a unique app name that helps users distinguish your version from others, such as "<Your Company Name> Chat". See our `Brand and Visual Design Guidelines <https://handbook.mattermost.com/operations/operations/company-processes/publishing/publishing-guidelines/brand-and-visual-design-guidelines#name-usage-guidelines>`__ in our Company Handbook for branding details.
+
+Deploying Your Custom App
+-------------------------
+
+You have a number of ways to deploy your custom app for your users to download to their mobile devices.
+
+Using an Enterprise App Store
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is the most common way for customers to distribute their apps, and it’s our recommended approach. Once your custom app is added to your own Enterprise App Store, your users can download it from the store directly or from an EMM catalog. 
+
+Alternatively, you can use an EMM provider to push the application to the user’s device, then use the AppConfig standard to enforce a selection of app-specific controls. See our `Manage App Configuration Using AppConfig <https://docs.mattermost.com/mobile/deploy-mobile-apps-using-emm-provider.html#manage-app-configuration-using-appconfig>`__ documentation for details.
+
+Using an EMM Provider
+~~~~~~~~~~~~~~~~~~~~~
+
+See our `Deployment Using an EMM Provider <https://docs.mattermost.com/mobile/deploy-mobile-apps-using-emm-provider.html>`__ documentation to learn more about deploying your custom Mattermost Mobile apps through an EMM provider.
+
+Using Public App Stores
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This is the same process Mattermost uses to make Mattermost Mobile apps available for everyone. It’s a good choice if you're looking to white-label your own Mobile apps to remove Mattermost branding. Before your app can be listed on the public app stores, you need to submit it to the public app stores for review and approval.
+
+As part of the submission process, you need to identify an update strategy that accounts for the release of new versions of Mattermost Mobile apps. 
+
+This update strategy should include:
+
+- Reviewing compatibility requirements.
+- Validating mobile app versions connecting to the server.
+- Updating Mattermost Server.
+- Updating Mattermost Mobile apps.
+
+We highly recommend you update your custom Mattermost Mobile apps to incorporate any security or service releases. Prior to distribution, check any compatibility requirements for the Mobile apps and the Mattermost server. Consult the `Mattermost Mobile App Changelog <https://github.com/mattermost/mattermost-mobile/blob/master/CHANGELOG.md>`__ and the `Mattermost Server Changelog <https://docs.mattermost.com/administration/changelog.html>`__ for details.
```

Comment:
the changelog has moved https://docs.mattermost.com/administration/mobile-changelog.html

Reply:
Thanks! Link has been updated accordingly.